angrmanagement/ui/dialogs/preferences.py

```diff
@@ -235,6 +259,9 @@ def save_config(self):
         for i in self._font_options:
             i.update()
 
+    def revert_unsaved(self):
```

Comment:
Identical to base class implemantation

Reply:
The base class raises `NotImpelementedError`.

Reply:
Oh, I guess I was looking at the wrong thing 

Why not have the default implementation do nothing instead of raising an error? Or just drop the method because there's not an implementation that does anything?